packages/react/src/components/Authenticator/VerifyUser/VerifyUser.tsx

```diff
@@ -60,6 +60,7 @@ export const VerifyUser = (): JSX.Element => {
     },
   } = useCustomComponents();
 
+  // TODO: expose unverifiedAttributes from `useAuthenticator`
   const { _state, isPending } = useAuthenticator();
```

Comment:
So in situations where you are pulling out the `_state` there is no context to be providedd? I noticed the same in the `FormFields.tsx` file too.

Reply:
Should we optimize for `isPending` here or would that prevent us from picking up changes in `_state`?

Reply:
@ErikCH Yep, `unverifiedAttributes` is not currently provided from our facade. We should to make codes simpler, but I'll do that in a separate PR. (And with conversations about refactoring facade values, we could use a discussion before that).

Reply:
@calebpollman you're right! I forgot to optimize this part.

I was initially torn here because `useAuthenticator((context) => [context.isPending])` wouldn't satisfy exhausive deps. But considering it's internal and that we're in so much re-rendering hell, we should make this optimization.

Reply:
Done here cabac4e3d16f57610fdc5d9431c871fc07d904de

Reply:
Oops wrong commit -- it should be 8fa81743bfbd41942fbc2b290b519ed51f5c2ce5